### PR TITLE
Tidy up spacing

### PR DESCRIPTION
**Pull request by afestein**
Tidy up of some minor spacing and extra lines for consistency and easier
reading.

### CODE REVIEW

**Comment by afestein** [comment]
👀 please @willem-h @jjjjesse

File: group2_assignment2_solution.ipynb [hunk 1]
```diff
@@ -273,7 +271,7 @@
     "                                             'lang':'en',\n",
     "                                             'result_type':'recent', #INSERT YOUR CODE\n",
     "                                             'max_id':max_id_str, #INSERT YOUR CODE\n",
-    "                                             'geocode':'{},{},{}mi'.format(AUS_LAT, AUS_LONG,AUS_RADIUS) #INSERT YOUR CODE\n",
+    "                                             'geocode':'{},{},{}mi'.format(AUS_LAT,AUS_LONG,AUS_RADIUS) #INSERT YOUR CODE\n",
```
Comment (willem-h) [inline group2_assignment2_solution.ipynb]: If anything could we add spaces to this?

Comment (afestein) [reply group2_assignment2_solution.ipynb]: There ya go 👌

File: group2_assignment2_solution.ipynb [hunk 2]
```diff
@@ -245,13 +245,16 @@
...
+    "    resp = api.request(\n",
+    "        'search/tweets',\n",
+    "        { \n",
+    "            'q': keyword,\n",
+    "            'count': batch_count,\n",
+    "            'lang': 'en',\n",
+    "            'result_type': 'recent',\n",
+    "            'geocode': '{},{},{}mi'.format(AUS_LAT, AUS_LONG, AUS_RADIUS)\n",
+    "        }\n",
+    "    )\n",
```
Comment (afestein) [inline group2_assignment2_solution.ipynb:257]: Me likey

**Comment by afestein** [comment]
I can’t approve this PR because it’s mine, so I'll leave it to you @willem or @jjjjesse

**Comment by willem-h** [comment]
Hope you don't mind @afestein 😉

**Comment by afestein** [comment]
Not at all

**Comment by willem-h** [comment]
I hadn't reloaded the page when I sent that comment ☝️  😂

**Review by willem-h — APPROVED** [review]

**Pull request merged**